### DISCUSSION TITLE
Is CsvRecord class deprecated ?

### DISCUSSION THREAD

**Discussion by ronee23** [osiegmar/FastCSV#95] [General]
The CsvRecord class is missing the below release.

Have the following gradle dependency,
implementation 'de.siegmar:fastcsv:2.2.2'

When i try to utilize the class, i dont find it.

![image](https://github.com/osiegmar/FastCSV/assets/88320014/2792007d-0146-4ea8-a685-3177aec2ba8c)

But the examples provided still uses CsvRecord class.

How can i run the example without CsvRecord ?

Cheers

**Comment by osiegmar** [comment]
The `CsvRecord` comes with version 3 (current in development in the master branch). If you're using version 2.2.2 please use documentation/examples from that branch: https://github.com/osiegmar/FastCSV/tree/v2.2.2
